src/components/schemas/fields.yaml

```diff
@@ -1507,3 +1507,18 @@ query_params:
   description: | 
     This field contains the query param string similar to the query params passed in GET call for displaying data in the list view.
   example: state=eq.PAID
+
+joined_at:
```

Comment:
we directly use timestamptz in employees, can use the same ref and just add description
```
  joined_at:
      $ref: './fields.yaml#/timestamptz_utc'

```